frameworks/kafka/build.gradle

```diff
@@ -22,7 +22,7 @@ ext {
     junitVer = "4.12"
     systemRulesVer = "1.16.0"
     mockitoVer = "2.10.0"
-    dcosSDKVer = "0.56.0"
+    dcosSDKVer = "0.55.5"
```

Comment:
Should be `0.56.1`

Reply:
yeah, I'm going to remove that sdk fix commit and rebase with master as now in master its fixed

Reply:
will do in the last moment to not to lose this PR review